KsApi/MockService.swift

```diff
@@ -17,16 +17,15 @@
 
     fileprivate let cancelBackingResult: Result<GraphMutationEmptyResponseEnvelope, GraphError>?
 
-    fileprivate let changeCurrencyResponse: GraphMutationEmptyResponseEnvelope?
-    fileprivate let changeCurrencyError: GraphError?
+    fileprivate let changeCurrencyResult: Result<UpdateUserEnvelope, ErrorEnvelope>?
 
-    fileprivate let changeEmailResult: Result<UpdateAccountEnvelope, ErrorEnvelope>?
+    fileprivate let changeEmailResult: Result<UpdateUserEnvelope, ErrorEnvelope>?
```

Comment:
I'm only consolidating the return value. `UpdateUserEnvelope` can be the returned type for both the `UpdateUserAccountMutation` and `UpdateUserProfileMutation`. Previously it was just called `UpdateAccountEnvelope`.

We can't really consolidate both mutations mentioned above because they are separate types of mutations on the GraphAPI. Tested this on GraphiQL. If you can of a way to do this though, I'm happy to consolidate them.

Reply:
The same return type is fine with me, just wanted to verify I knew what was going on with the change.